vendor/assets/javascripts/util.js

```diff
@@ -0,0 +1,7 @@
+Cmsify.util = {
+  filterAuthenticityToken: function(arr) {
+    return arr.filter(function(element) {
+      return element.name !== "authenticity_token";
+    });
+  }
+}
```

Comment:
I guess I'm not clear on why one case would need it and another not. Why are you deleting it here?